parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -29,6 +29,10 @@ cleanupContainerdDlFiles() {
     rm -rf $CONTAINERD_DOWNLOADS_DIR
 }
 
+removeApport() {
+    sudo apt-get purge apport
```

Comment:
I would move this to https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSUbuntu/gen2/1804/2022.08.02.txt#L212-L213

similar to https://github.com/Azure/AgentBaker/blob/4da8de64fe542bd849a62cc7625e0b5fa487f00d/vhdbuilder/packer/install-dependencies.sh#L556-L562 and conditioned on ubuntu os

preferably before this line so the package output is right -__- https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/install-dependencies.sh#L518